tests/README.md

```diff
@@ -155,6 +155,15 @@ exit_code, output = ctrl.run_python("python/monitor.py")
 
 A full example of how to use the python bindings can be found in the [monitor open-close test](./tests/tier0/monitor-open-close/).
 
+### Generating test ID
+
+```shell
+cd ~/bluechi/tests
+tmt test id .
+New id 'UUID' added to test '/tests/path_to_your_new_test'.
+...
+```
```

Comment:
Hmm, shouldn't the formatting look like below?

```shell
cd ~/bluechi/tests
tmt test id .
```

New id 'UUID' is added to the new test.

Reply:
New id 'UUID' is added to the new test. 
This is return in CLI from tmt command

Reply:
Ah, OK, now it makes sense to me